coincurve/__init__.py

```diff
@@ -1,6 +1,41 @@
-from coincurve.context import GLOBAL_CONTEXT, Context
-from coincurve.keys import PrivateKey, PublicKey, PublicKeyXOnly
-from coincurve.utils import verify_signature
+def load_secp256k1_conda_library():
+    """Load the secp256k1 library from the conda environment."""
+    import warnings
+
+    from coincurve._secp256k1_library_info import SECP256K1_LIBRARY_NAME, SECP256K1_LIBRARY_TYPE
+
+    if SECP256K1_LIBRARY_TYPE != 'EXTERNAL':
```

Comment:
Isn't internal the expected default and so this would happen for most users?

Reply:
Right, it should only do extra when coincurve was built with a system library. I need to clean up my debugging.
I am off for the next 10hrs

Reply:
Let me take another look and use `ctypes.util.find_library()` and only add conda when the library is not found, then fail if it is not a conda environment

Reply:
I reorganized it a little (it works correctly well on my testing branch). I added the build with conda for verification. I am working on an attempt to test with a macos build, if that works, I guess the linux build should also be manageable. The idea would be to install conda on the docker VM used for  the build in cibuildwheel